pkgs/top-level/aliases.nix

```diff
@@ -38,6 +38,20 @@ mapAliases ({
   accounts-qt = libsForQt5.accounts-qt; # added 2015-12-19
   adobeReader = adobe-reader; # added 2013-11-04
   adobe_flex_sdk = apache-flex-sdk; # added 2018-06-01
+  adoptopenjdk-hotspot-bin-17 = throw "AdoptOpenJDK is now Temurin. Use temurin-bin-17"; # added 2021-10-03
+  adoptopenjdk-jre-hotspot-bin-17 = throw "AdoptOpenJDK is now Temurin. Use temurin-jre-bin-17"; # added 2021-10-03
+  adoptopenjdk-hotspot-bin-15 = throw "AdoptOpenJDK 15 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-jre-hotspot-bin-15 = throw "AdoptOpenJDK 15 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-openj9-bin-15 = throw "AdoptOpenJDK 15 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-jre-openj9-bin-15 = throw "AdoptOpenJDK 15 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-hotspot-bin-14 = throw "AdoptOpenJDK 14 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-jre-hotspot-bin-14 = throw "AdoptOpenJDK 14 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-openj9-bin-14 = throw "AdoptOpenJDK 14 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-jre-openj9-bin-14 = throw "AdoptOpenJDK 14 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-hotspot-bin-13 = throw "AdoptOpenJDK 13 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-jre-hotspot-bin-13 = throw "AdoptOpenJDK 13 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-openj9-bin-13 = throw "AdoptOpenJDK 13 is no longer maintained"; # added 2021-10-03
+  adoptopenjdk-jre-openj9-bin-13 = throw "AdoptOpenJDK 13 is no longer maintained"; # added 2021-10-03
```

Comment:
We don't add aliases for all version variants otherwise the aliases file would be to big.